wgsl/index.bs

```diff
@@ -3581,6 +3605,17 @@ The `for(initializer; condition; continuing) { body }` statement is syntactic su
 * If `condition` is non-empty, it is checked at the beginning of the loop body and if unsatisfied then a [[#break-statement]] is executed.
 * If `continuing` is non-empty, it becomes a [[#continuing-statement]] at the end of the loop body.
 
+The `initializer` of a for loop is executed once prior to executing the loop.
```

Comment:
I wonder if, instead of properly specifying this like this text is trying to do, we should just write pseudo-code for how `for` translates into `loop` and call it a day.

Reply:
The 'for statement' section kind of does do that.  But it's a bit imprecise, and leaves the impression that the scope of a thing declared in the  body is extends to the "update" clause of the for loop. Which it doesn't.
So better to spell it out.

But yes, this could be revisitec.  (And, see for loops are more complicated than you remember.)